databricks_cli/clusters/cli.py

```diff
@@ -248,6 +248,23 @@ def spark_versions_cli(api_client):
     click.echo(pretty_format(ClusterApi(api_client).spark_versions()))
 
 
+@click.command(context_settings=CONTEXT_SETTINGS)
+@click.option('--cluster-id', required=True, type=ClusterIdClickType(),
+              help=ClusterIdClickType.help)
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def permanent_delete_cli(api_client, cluster_id):
+    """
+    Permanently deletes a Spark cluster.
```

Comment:
Nanonit: I know we copied this from the [REST API docs](https://docs.databricks.com/api/latest/clusters.html#permanent-delete) - we could consider explicitly mentioning that this is irreversible (although I guess "permanent" already implies that). Also, maybe we should say "Databricks cluster" instead of "Spark cluster" to maintain parity with the other clusters commands, e.g:

"Permanently and irreversibly deletes a Databricks cluster"